hatice-ecevit-2016400138/assignment1/KthToLast.java

```diff
@@ -0,0 +1,43 @@
+
+public class KthToLast {
+    public static void main(String[] args) {
+        Node<Integer> n1 = new Node<>(3);
+        Node<Integer> n2 = new Node<>(8, n1);
+        Node<Integer> n3 = new Node<>(9, n2);
+        Node<Integer> n4 = new Node<>(7, n3);
+        Node<Integer> n5 = new Node<>(5, n4);
+        Node<Integer> n6 = new Node<>(2, n5);
+        Node<Integer> n7 = new Node<>(0, n6);
+
+        System.out.println(kthToLastElement(n7, 3).getData());
+
+    }
+
+    /**
+     * @param head:head of the given linked list
```

Comment:
nit: do not use `:` after the name of the param. The javadoc tool does not like that.